pkg/handlers/generic/lifecycle/csi/aws-ebs/handler.go

```diff
@@ -135,7 +142,12 @@ func (a *AWSEBS) createStorageClasses(ctx context.Context,
 		if setAsDefault {
 			sc.ObjectMeta.Annotations = defaultStorageClassMap
 		}
-		if err := client.ServerSideApply(ctx, a.client, &sc); err != nil {
+		workloadClient, err := remote.NewClusterClient(ctx, "", a.client, capiutil.ObjectKey(&cluster))
+		if err != nil {
+			return err
+		}
+
+		if err := client.ServerSideApply(ctx, workloadClient, &sc); err != nil {
```

Comment:
Why not apply this via CRS?

Reply:
how were you thinking this would look?

Reply:
I think to be aligned with other addons is all.

Reply:
Working on this.